Resources/views/Block/block_search_result.html.twig

```diff
@@ -12,43 +12,44 @@ file that was distributed with this source code.
 {% extends sonata_block.templates.block_base %}
 
 {% block block %}
-    <div class="col-md-3">
-        <div>
-            <h4>
-                {% if admin.hasRoute('list') %}
-                    <a href="{{ admin.generateUrl('list') }}">{{ admin.label|trans({}, admin.translationdomain) }}</a>
-                {% else %}
+    <div class="col-lg-4 col-md-6">
+        <div class="box">
+            <div class="box-header with-border">
+                <h3 class="box-title">
+                    {% set icon = settings.icon|default('') %}
+                    {{ icon|raw }}
                     {{ admin.label|trans({}, admin.translationdomain) }}
-                {% endif %}
-
+                </h3>
                 {% if pager and pager.getNbResults() > 0 %}
```

Comment:
According to [AdminLTE documentation](https://almsaeedstudio.com/themes/AdminLTE/documentation/index.html#component-box), it should be:

```twig
<div class="box-tools pull-right">
    {% if pager and pager.getNbResults() > 0 %}
        <span class="badge">{{ pager.getNbResults() }}</span>
    {% elseif admin.hasRoute('create') and admin.isGranted('CREATE') %}
        <a href="{{ admin.generateUrl('create') }}" class="btn btn-box-tool">
            <i class="fa fa-plus"></i>
        </a>
    {% endif %}
</div>
```

And I think create button could be here even if search has some results.